openmetadata-ui/src/main/resources/ui/cypress/common/CustomizeLandingPageUtils.ts

```diff
@@ -93,7 +93,7 @@ export const navigateToLandingPage = () => {
 export const openAddWidgetModal = () => {
   interceptURL(
     'GET',
-    `/api/v1/docStore?fqnPrefix=KnowledgePanel`,
+    `/api/v1/docStore?fqnPrefix=KnowledgePanel&limit=25`,
```

Comment:
instead of adding limit add * so in future anything change there, test do not fail